test/integration/alias.test.js

```diff
@@ -36,12 +36,23 @@ beforeEach(async (done, t) => {
         key: 'change_me',
     }).run();
 
-    const publishCmd = `${eik} npm scroll-into-view-if-needed 2.2.24
-        --token ${token}
-        --server ${address}
-        --cwd ${folder}`;
+    const assets = {
+        name: 'scroll-into-view-if-needed',
```

Comment:
Oh wow, who uses `scroll-into-view-if-needed` in real apps?? *Cringe* 😉

Reply:
I heard it's a popular module by a rock star programmer. ;)